### DISCUSSION TITLE
Concept for splitting audit logs as file

### DISCUSSION THREAD

**Discussion by MarcoJanecki** [cerbos/cerbos#1765] [Q&A] [answered]
Hey folks,

we are curious if there are any mechanisms (or if not - ideas) how to tackle the problem of an increasing log file size of the audit log when a deployment runs many months. 

- Is Cerbos somehow splitting the log file by its own?
- Should we shift to a new log file every month or so?

We have configured our deployment to output the audit logs to a file `/path/to/audit.log` which me mounted to the Pod from an Azure Storage Account FileShare.

Any thoughts are appreciated. :)

**Comment by charithe** [comment, marked as answer]
Hi. Cerbos doesn't currently roll audit logs. We are hoping to implement it at some point (https://github.com/cerbos/cerbos/issues/1758). Until then, our advice is to use an external log rotation tool such as logrotate to manage that process.

**Reply by MarcoJanecki** [reply]
Thanks, we are looking forward to such feature and will tackle that via Kubernetes mechanisms for now. :)
